h2o-py/tests/testdir_algos/gbm/pyunit_gbm_wendy_problem.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import division
+from builtins import range
+import sys
+
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils, assert_equals
+from h2o.estimators import H2OGradientBoostingEstimator, H2OXGBoostEstimator
+from h2o.tree import H2OTree
+
+
+def assert_list_equals(expected, actual, delta=0):
+    assert_equals(len(expected), len(actual))
+    for i in range(0, len(expected)):
+        assert_equals(expected[i], actual[i], delta=delta)
+
+
+def gbm_predict_contributions_sorting_large():
+    fr = h2o.import_file(pyunit_utils.locate("bigdata/laptop/covtype/covtype.full.csv"))
```

Comment:
can you change the test name gbm_predicton_contributions_sorting_large to something that you are investigating.  A nitpik.